legacy/.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  root: true,
```

Comment:
After updating `ecmaVersion`, eslint complains with this deprecation warning:

```
(node:51996) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "../../../../../.eslintrc.json")
```